sdk/alertsmanagement/Microsoft.Azure.Management.AlertsManagement/src/Generated/Models/ErrorResponse1.cs

```diff
@@ -0,0 +1,60 @@
+// <auto-generated>
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for
+// license information.
+//
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is
+// regenerated.
+// </auto-generated>
+
+namespace Microsoft.Azure.Management.AlertsManagement.Models
+{
+    using Newtonsoft.Json;
+    using System.Linq;
+
+    /// <summary>
+    /// Describe the format of an Error response.
+    /// </summary>
+    public partial class ErrorResponse1
```

Comment:
There must be something wrong with the spec to generate such a wired named class. Could you point to me the openapi spec review PR(s)?

Reply:
Hi @isra-fel, thanks for the review.

I didn't really change that part of the spec. This is the PR with the changes I made: 
https://github.com/Azure/azure-rest-api-specs/pull/9540

Reply:
I got it, AlertsManagement SDK was generated a long time ago and it was only based on AlertsManagement.json. But now the default tag is pointing to both AlertsManagement.json and SmartDetectorAlertRulesApi.json.

I don't suppose that's what you want because your PR only touched the former one, so this is my suggestion: edit `generate.ps1` and add a param `-ConfigFileTag package-preview-2019-05` -- [this tag](AlertsManagement.json) only contains AlertsManagement.json

Reply:
@isra-fel , yes I see what you mean. Thanks, generated again with the relevant tag.